config/application.rb

```diff
@@ -18,6 +18,18 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+ActiveRecord::AssociatedObject.class_eval {
```

Comment:
Does this need to live in `config/application.rb`? Or could this be in an initializer/concern/superclass somewhere?

Reply:
Yeah, I thought about that and that could work too. I just pushed a commit putting this in an initializer.